2_isolation/README.md

```diff
@@ -8,17 +8,24 @@ In addition we were asked to develop our own heuristics to attempt to beat the s
 
 ## Instructions
 
-Play against AI:
+* Switch to Python 3.6 or greater environment
+
+* Install necessary dependencies:
+```bash
+$ pip install termcolor
```

Comment:
I agree with your suggestion, but I've included a link to the "main" README.md incase users overlook it and just directly go to 2_isolation/README.md (like I did).

FYI, I had trouble using pipenv myself due to conflicts, but I created an [Pipenv issue](https://github.com/kennethreitz/pipenv/issues/288) and working through trying to resolve it